tests/unittests/OperatorTest.cpp

```diff
@@ -3362,6 +3362,66 @@ TEST_P(InterpAndCPU, insertTensorTest) {
   }
 }
 
+/// Test RowwiseQuantizedFullyConnected Node.
+TEST_P(InterpOnly, rowWiseFCTest) {
+  // In this test we subtract the outputs of a row-wise quantized FC and a
+  // floating-point FC and ensure that the error is below some low value.
+  auto *input = mod_.createVariable(ElemKind::FloatTy, {1, 10, 10, 3}, "in");
+  auto *fc = F_->createFullyConnected("FC", input, 30);
+  auto *res =
+      mod_.createVariable(ElemKind::FloatTy, fc->getResult().dims(), "res");
+
+  auto weights = fc->getWeights();
+  auto bias = fc->getBias();
+
+  input->getPayload().getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  llvm::cast<Variable>(bias)->getPayload().getHandle().randomize(
+      0, 0.00001, mod_.getPRNG());
```

Comment:
Why is bias so small? I have a feeling that such small value doesn't allow for good testing. Does the test still pass if you remove bias-related code (`sum += B`) from Interpreter implementation?

Reply:
I copied this settings from IntFC in OperatorTest.cpp. Will check this bias.